k3os/install.sh

```diff
@@ -13,6 +13,7 @@ get_url()
 {
     FROM=$1
     TO=$2
+    echo "Fetching ${FROM} to ${TO}..."
```

Comment:
If a change in the k3os directory is needed, put it in a separate commit

Reply:
Update in https://github.com/rancher/harvester-installer/pull/33/commits/126f269b0e1a3fa67efcc4363c344d710fe68dc5